gateways/istio-ingress/templates/certificate.yaml

```diff
@@ -42,3 +42,26 @@ spec:
       - {{ $key }}
 {{- end }}
 {{- end }}
+
+{{ if .Values.gateways.kiali_gateway.enabled }}
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: telemetry-gw-cert
+  namespace: {{ .Release.Namespace }}
+spec:
+  secretName: telemetry-gw-cert
+  issuerRef:
+    name: letsencrypt
+    kind: ClusterIssuer
+  commonName: {{ .Values.gateways.kiali_gateway.domain_name}}
+  dnsNames:
+    - {{ .Values.gateways.kiali_gateway.domain_name}}
```

Comment:
I would prefer if domain_name was a common setting, but we can fix this later.

Reply:
Are you suggesting a common setting for all telemetry addons? If so, I agree that makes more sense